Makefile

```diff
@@ -32,6 +32,13 @@ install:
 	${GO_BUILD_ENVVARS} go install \
 		-ldflags "-X main.version=${VERSION} -X main.commitHash=${COMMIT_HASH}"
 
+format:
+# Add new paths        find . \( -path './vendor' -o -path <path> \) -prune -o -type f -iname '*.go' -print
```

Comment:
not sure why the comment has the "    find ...." text in it ??

Is this supposed to be a comment for the line below it? The comment is not indented over, so I'm not sure what this comment is telling me. Add new paths??

Reply:
It is a comment about how to add a new paths to the script

Reply:
OK, I would just add a bit more description here. I think what this is saying is we can remove the words "Add new paths" and replace `<path>` with a path containing .go files and the line will format those .go files in the given path. But even that doesn't seem right because it doesn't actually do gofmt. Unless this is saying add this with a for-loop, too?

In other words, this comment is very confusing and/or incomplete :) Either make the comments more descriptive of what we have to do or just delete it entirely to avoid confusion. It's OK to have multiple line comments ;)

Reply:
Sorry I was wrong ^^this is for exclude folders like /vendor/ We don't want to fix gofmt errors of others packages .